src/core/examples.js

```diff
@@ -58,7 +59,20 @@ function makeTitle(conf, elem, num, report) {
   `;
 }
 
-export function run(conf) {
+/**
+ * Link <a href="#example">
+ */
+function replaceEmptyAnchors() {
+  for (const id of examplesMap.keys()) {
```

Comment:
`for (const [id, text] of examplesMap.entries()) {` could be better, which can replace `examplesMap.get(id)` with `text`.

Reply:
good suggestion!

Reply:
A little late. We don't need to do `.entries()` 😅

Reply:
Oh, I didn't know about this! Thanks!